### ISSUE TITLE
Referenced version of ESP-IDF not available

### ISSUE THREAD

**Issue by jorgie0** [Xinyuan-LilyGO/LilyGo-Display-IDF#5]
Hi
The referenced version of ESP-IDF is not available for download.

`The example relies on esp-idf V5.3.0, Please see the [Getting Started Guide](https://docs.espressif.com/projects/esp-idf/en/latest/get-started/index.html) for full steps to configure and use esp-idf to build projects.`

**Comment by lewisxhe** [comment]
There is no error in the link, what are you referring to?

5.3.0 is currently in the master branch

**Comment by jorgie0** [comment]
@lewisxhe  Then I am missing something I cannot see V5.3 of ESP IDF.
I am looking here:
https://github.com/espressif/esp-idf/tree/master

![esp idf git](https://github.com/Xinyuan-LilyGO/LilyGo-Display-IDF/assets/5963000/fb92ed02-90cf-4b29-aed9-ee6b7806180e)

**Comment by lewisxhe** [comment]
![image](https://github.com/Xinyuan-LilyGO/LilyGo-Display-IDF/assets/22990954/4efe13da-b96d-444c-9b50-d4af9e7ab01e)

**Comment by jorgie0** [comment]
@lewisxhe my point is that version 5.3 cannot be found where the red arrow is pointed. The paragraph I quoted stated that it relies on esp-idf V5.3.0,
So please help me understand what I am doing wrong? 
![esp idf git2](https://github.com/Xinyuan-LilyGO/LilyGo-Display-IDF/assets/5963000/7f1fde0f-2d1a-4029-ae2c-ed56569fea75)

**Comment by krystalmonolith** [comment]
@jorgie0 You are correct ... As of now (March 19 2024) I see no "5.3.x" tag or release of the SDK... The last one I see is 5.2.1 on their tags page at https://github.com/espressif/esp-idf/tags . There have been many commits since then and I assume @lewisxhe intends that you should use them by working from the ```master``` branch, which will  eventually become 5.3.x .

**Comment by lewisxhe** [comment]
Yes, it currently belongs to the master branch. Of course, you can use v5.x.

**Comment by Omnomios** [comment]
I used the following commands to get ESP-IDF 5.3
```
git pull --all
git checkout remotes/origin/release/v5.3
```
Then run the install.sh as normal.

Might be helpful to someone.
